Terminal.Gui/Text/TextFormatter.cs

```diff
@@ -170,6 +170,13 @@ public TextDirection Direction
         }
     }
 
+    /// <summary>
+    ///     Determines if the bounds width will be used or only the text width will be used,
+    ///     If <see langword="true"/> all the bounds area will be filled with whitespaces and the same background color
+    ///     showing a perfect rectangle.
+    /// </summary>
+    public bool FillRemaining { get; set; }
```

Comment:
I'm not convinced this property is needed. 

Why not just have code that needs the fillRemaining parameter to be true pass true?